CONTRIBUTING.md

```diff
@@ -1,66 +1,67 @@
 <h2 align="center">Contributing</h2>
 
-<p align="center">Contributing to Dark Reader can be in a variety of ways. Here, you can learn more about it. Thank you in advance for reading this <code>CONTRIBUTING.md</code>.</p>
+<p align="center">You can contribute to Dark Reader in a variety of ways. Here, you can learn more about it.
+Thank you in advance for reading this <code>CONTRIBUTING.md</code></p>
 
 ## Sponsor
 
-<a href="https://opencollective.com/darkreader/donate" target="_blank" rel="noreferrer noopener"> <img src="https://opencollective.com/darkreader/donate/button@2x.png?color=blue" width=300 /></a>
+Sponsor the development of Dark Reader by clicking here:
 
-Sponsor the development of Dark Reader
+<a href="https://opencollective.com/darkreader/donate" target="_blank" rel="noreferrer noopener"> <img src="https://opencollective.com/darkreader/donate/button@2x.png?color=blue" width=300 /></a>
 
-## Translation
+## Translate
```

Comment:
I think `Translation` is better as `Translate` is a verb and should be used in a sentence-like way.

Reply:
Either is good by me.  I wanted to encourage action, so I used verbs.  Also, the first item is "Sponsor", which is being used as a verb, so I wanted to be consistent for all the headings.

Reply:
Well then you can change `Sponsor` to `Sponsoring` or something alike.

Reply:
You *really* don't like verbs?